qiskit/transpiler/passes/calibration/rzx_builder.py

```diff
@@ -343,36 +332,64 @@ def _filter_comp_tone(time_inst_tup):
     return False
 
 
-def _check_calibration_type(cx_sched) -> Tuple[CXCalType, List[Play], List[Play]]:
+def _check_calibration_type(
+    inst_sched_map: InstructionScheduleMap, qubits: List[int]
+) -> Tuple[CRCalType, List[Play], List[Play]]:
     """A helper function to check type of CR calibration.
 
     Args:
-        cx_sched: A target schedule to stretch.
+        inst_sched_map: instruction schedule map of the backends
+        qubits: ordered tuple of qubits for cross resonance (q_control, q_target)
 
     Returns:
         Filtered instructions and most-likely type of calibration.
 
     Raises:
         QiskitError: Unknown calibration type is detected.
     """
+    cal_type = None
+    if "cx" in inst_sched_map.qubit_instructions(qubits):
```

Comment:
```suggestion
    if inst_sched_map.has("cx", qubits):
```
nitpick but you have dedicated method. same for others.

Reply:
Good point, changed to dedicated method.